src/libsyntax/ast.rs

```diff
@@ -2640,28 +2600,43 @@ impl ItemKind {
     }
 }
 
-pub type ForeignItem = Item<ForeignItemKind>;
+pub type ForeignItem = Item<AssocItemKind>;
+pub type ForeignItemKind = AssocItemKind;
```

Comment:
Could you add a FIXME comment telling that these definitions will be "unmerged"?

Reply:
Done.